src/Avalonia.Visuals/Point.cs

```diff
@@ -267,5 +267,24 @@ public Point WithY(double y)
         {
             return new Point(_x, y);
         }
+
+        /// <summary>
+        /// Deconstructor for decomposition Point
```

Comment:
Could do with a `.` at the end. Personally I'd prefer something like "Deconstructs the point into its X and Y components.", but I've not checked the rest of the code-base with respect to consistency.

(Similarly with the other deconstructors)